### PR TITLE
Retrofit

### PR DESCRIPTION
**Pull request by danilkolikov**
Skeleton of retrofit for VK done

### CODE REVIEW

File: app/src/main/java/ly/loud/loudly/application/Loudly.java [hunk 1]
```diff
@@ -116,6 +102,15 @@ public synchronized static Activity getCurrentActivity() {
         return Loudly.activity;
     }
 
+    /**
+     * Store current visible activity into Loudly
+     *
+     * @param activity current activity
+     */
+    public synchronized static void setCurrentActivity(Activity activity) {
```
Comment (ZeRoGerc) [inline app/src/main/java/ly/loud/loudly/application/Loudly.java]: Nullability

File: app/src/main/java/ly/loud/loudly/application/models/VKModel.java [hunk 2]
```diff
@@ -105,124 +91,217 @@ private void loadFromDB() {
...
+    private ly.loud.loudly.base.attachments.Attachment toAttachment(@NonNull Attachment attachment) {
+        Photo photo = attachment.photo;
+        if (photo != null) {
+            return new Image(attachment.photo.photo604, new Point(photo.width, photo.height),
+                    Networks.VK, new Link(photo.id));
+        }
+        return null;
+    }
+
+    private void setAttachments(@NonNull ly.loud.loudly.base.says.Say say, @NonNull Say loaded) {
```
Comment (ZeRoGerc) [inline app/src/main/java/ly/loud/loudly/application/models/VKModel.java]: import

Comment (danilkolikov) [inline app/src/main/java/ly/loud/loudly/application/models/VKModel.java]: that's okay

**Comment by ZeRoGerc** [comment]
ok

**Pull request merged**